code/game/gamemodes/blob/blobs/shield.dm

```diff
@@ -5,7 +5,7 @@
 	desc = "A solid wall of slightly twitching tendrils."
 	health = 150
 	maxhealth = 150
-	brute_resist = 0.1
+	brute_resist = 0.25

```

Comment:
Normal blob tiles have 25 health(if in pulse range) and 21 otherwise, and have 0.25% brute resist until they hit 10 health, where it goes back to 0.5%
So shield blobs are still super-resistant to brute compared to normal blobs, but not ~basically indestructible levels of resistant; one shield blob would have effective 1500 health versus brute, this drops that to effective 600 which is frankly still goddamn high.